dist/docker/debian/build_docker.sh

```diff
@@ -45,18 +51,18 @@ while [ $# -gt 0 ]; do
     esac
 done
 
-container="$(buildah from docker.io/centos:7)"
+container="$(buildah from docker.io/ubuntu:20.04)"
 
 packages=(
-    "build/dist/$mode/redhat/RPMS/$arch/$product-$version-$release.$arch.rpm"
-    "build/dist/$mode/redhat/RPMS/$arch/$product-server-$version-$release.$arch.rpm"
-    "build/dist/$mode/redhat/RPMS/$arch/$product-conf-$version-$release.$arch.rpm"
-    "build/dist/$mode/redhat/RPMS/$arch/$product-kernel-conf-$version-$release.$arch.rpm"
-    "build/dist/$mode/redhat/RPMS/$arch/$product-node-exporter-$version-$release.$arch.rpm"
-    "tools/java/build/redhat/RPMS/noarch/$product-tools-$version-$release.noarch.rpm"
-    "tools/java/build/redhat/RPMS/noarch/$product-tools-core-$version-$release.noarch.rpm"
-    "tools/jmx/build/redhat/RPMS/noarch/$product-jmx-$version-$release.noarch.rpm"
-    "tools/python3/build/redhat/RPMS/$arch/$product-python3-$version-$release.$arch.rpm"
+    "build/dist/$mode/debian/$product\_$version-$release-1_$arch.deb"
```

Comment:
Why this backslash? Is it to unconfuse "$product"? Better to use ${product}, it's the standard way.

Reply:
Changed.